### DISCUSSION TITLE
Question: Where can I get Admin account and relayer account details. [Newbie, Unclear from docs]

### DISCUSSION THREAD

**Discussion by spunkyshooter** [0xPolygon/polygon-edge#450] [Q&A] [answered]
I am trying to setup ChainBridge 

I was following doc: https://edge-docs.polygon.technology/docs/additional-features/chainbridge/deploy-contracts

```
# Deploy all required contracts into Polygon PoS chain
$ cb-sol-cli deploy --all --chainId 99 \
  --url https://rpc-mumbai.matic.today \
  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
  --relayers [RELAYER_ACCOUNT_ADDRESS] \
  --relayerThreshold 1
```

Where can I get these details below ?
`ADMIN_ACCOUNT_PRIVATE_KEY`
`RELAYER_ACCOUNT_ADDRESS`
`RELAYER_ACCOUNT_PRIVATE_KEY`

These are my assumptions:
- I can use the metamask wallet account as admin account and use its private key.
- For Relayer, I need to create a new account in metamask and use.

**Comment by Kourin1996** [comment, marked as answer]
@spunkyshooter Hi, Thank you for your questions. 

You can any accounts you want to use for admin and relayer account. If you're using Metamask, you can get the private key by exporting in account details. 

The address of the private key which you gave when you run `cb-sol-cli deploy` command will be assigned `admin` role as default. This `admin` role is required to run some specific method call in bridge like registering resource, chainging fee and setting burnable.

On the other hand, relayer account is needed to run relayer. Only allowed relayer accounts can give proposals and votings to bridge contract from the signal in source chain. `--relayers` in `cb-sol-cli deploy` command gives bridge initial relayer addresses. You need to give same address you specified there to `config.json` for relayer configuration.

Please check the below link for more details.
https://edge-docs.polygon.technology/docs/additional-features/chainbridge/requirements

**Comment by zivkovicmilos** [comment]
@Kourin1996 

Converting this to a discussion
